addons/account/tests/test_tour.py

```diff
@@ -1,28 +1,10 @@
 # Part of Odoo. See LICENSE file for full copyright and licensing details.
-
-from odoo import Command
+from odoo.addons.account.tests.common import AccountTestInvoicingHttpCommon
 import odoo.tests
 
 
 @odoo.tests.tagged('post_install_l10n', 'post_install', '-at_install')
-class TestUi(odoo.tests.HttpCase):
+class TestUi(AccountTestInvoicingHttpCommon):
 
     def test_01_account_tour(self):
-        # Reset country and fiscal country, so that fields added by localizations are
-        # hidden and non-required, and don't make the tour crash.
-        # Also remove default taxes from the company and its accounts, to avoid inconsistencies
-        # with empty fiscal country.
-        self.env.company.write({
-            'country_id': None, # Also resets account_fiscal_country_id
-            'account_sale_tax_id': None,
-            'account_purchase_tax_id': None,
-        })
-        account_with_taxes = self.env['account.account'].search([('tax_ids', '!=', False), ('company_id', '=', self.env.company.id)])
-        account_with_taxes.write({
-            'tax_ids': [Command.clear()],
-        })
-        # This tour doesn't work with demo data on runbot
-        all_moves = self.env['account.move'].search([('move_type', '!=', 'entry')])
-        all_moves.button_draft()
-        all_moves.with_context(force_delete=True).unlink()
```

Comment:
> [IMP] account: make tour inherit our test 

Why is this changing?
Have you tried the single l10n build?


<sub>I would be really glad if we could remove this though</sub>

Reply:
@william-andre This doesn't make sense to have this running for all localizations because some of them needs extra things  by the way.

Reply:
This is the test for the onboarding tour, that is used in every l10n. Whether you want it or not, it **has to** run for every l10n.

Reply:
This change is not necessary for this branch, we can drop the commit.

Reply:
@william-andre If it's run for the onboarding tour, I agree with you then

Reply:
commit dropped